.woodpecker/.phpunit.yml

```diff
@@ -38,7 +38,7 @@ steps:
   composer_install:
     image: friendicaci/php${PHP_MAJOR_VERSION}:php${PHP_VERSION}
     commands:
-      - git clone https://git.friendi.ca/friendica/friendica-addons.git addon
+      - git clone -b develop --single-branch https://git.friendi.ca/friendica/friendica-addons.git addon
```

Comment:
Why specifically the `develop` branch?

Reply:
The [previous commit has failed](https://ci.friendi.ca/repos/2/pipeline/2732/63) and it looks like that there was a wrong branch checked out. `git clone` checkouts the current `HEAD`, but for friendica `develop` branch we also need the friendica-addons `develop` branch. Thats why